charts/ComparisonLineGenerator.ts

```diff
@@ -8,19 +8,24 @@ export function generateComparisonLinePoints(
     xScaleType: string,
     yScaleType: string
```

Comment:
nit: do we have an enum type somewhere of "log" | "linear"? Might be helpful in the future if we look for all the places using log.

Reply:
Turns out we actually have that type! It's not an enum (yet), but can still use that.

Would probably be good to make that an actual TS enum, but that's a story for another time since it touches a lot of files.